backend/src/sdk.cpp

```diff
@@ -228,6 +362,40 @@ restinio::request_handling_status_t SDK::handleWebSocketSDKCall(
                 == message->opcode()) {
                 // Close connection
                 this->pWsRegistry.erase(wsh->connection_id());
+            } else if (restinio::websocket::basic::opcode_t::text_frame == message->opcode()) {
+                std::string payload = message->payload();
+
+                try {
```

Comment:
Can you put this in a separate function?

Reply:
Done!